### ISSUE TITLE
Could you please provide example code and launching command to few-shot finetune the model? 

### ISSUE THREAD

**Issue by withchencheng** [slei109/PATNet#37]
Hello authors,

Could you please provide example code and launching command to few-shot finetune the model? 
The correct way to use your code should be:
1. running `train.py` on pascal;
2. fine-tune on the few-shot support set; (Is the code for this step missing? Calling `finetune_reference()` function in Line 139 in `patnet.py`)
3. running `test.py` on query set of the target domain.
Am I correct?

Thank you.

**Comment by withchencheng** [comment]
Dear author,

Thank you for sharing this great work with us. I have a detailed question about the `finetune_reference()` function  in `patnet.py`.

Why is there an if statement for summing the kl loss (https://github.com/slei109/PATNet/blob/1d4c93343e4d18d2b2c8ffa4b3a9665da4dfac23/model/patnet.py#L155 ? Does this mean when we are fine-tuning on 5-shot, we just ignore the 4-th and 5-th shot for summing kl loss?

Thank you for your reply.

**Comment by withchencheng** [comment]
Dear authors,

I am confused by this if statement in the `finetune_reference()` function in patnet.py:
https://github.com/slei109/PATNet/blob/1d4c93343e4d18d2b2c8ffa4b3a9665da4dfac23/model/patnet.py#L155

Why do we only need to sum the first 3 kl losses?

Thank you for your reply.

**Comment by slei109** [comment]
Fine-tune step is optional. You can evaluate the model directly without fine-tuning it, like in test.py. If you'd like to boost the performance, you can call finetune_reference to fine-tune it. As explained on page 14, we found that fine-tuning the anchor layer of the low feature map achieves the best performance. We adopt fine-tuning the anchor layer for low dimensions to report all the experiment results.
